src/components/PostPageDetail/index.js

```diff
@@ -53,8 +52,7 @@ const PostPageDetailIdComponent = (props) => {
     navigateToReplyView,
     contextSource = CONTEXT_SOURCE.FEEDS,
     haveSeeMore,
-    parentData,
-    isKeyboardOpen
+    parentData
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `isKeyboardOpen` prop is removed. Ensure that this prop is not used anywhere else in the component or its children. If it is used, you might need to find an alternative way to get this information.
```diff
-    isKeyboardOpen
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->